data/data/azurestack/bootstrap/main.tf

```diff
@@ -0,0 +1,139 @@
+locals {
+  bootstrap_nic_ip_v4_configuration_name = "bootstrap-nic-ip-v4"
+  description                            = "Created By OpenShift Installer"
+  tags = merge(
+    {
+      "kubernetes.io_cluster.${var.cluster_id}" = "owned"
+    },
+    var.azure_extra_tags,
+  )
+}
+
+provider "azurestack" {
+  arm_endpoint    = var.azure_arm_endpoint
+  subscription_id = var.azure_subscription_id
+  client_id       = var.azure_client_id
+  client_secret   = var.azure_client_secret
+  tenant_id       = var.azure_tenant_id
+}
+
+resource "azurestack_storage_container" "ignition" {
+  name                  = "ignition"
+  resource_group_name   = var.resource_group_name
+  storage_account_name  = var.storage_account.name
+  container_access_type = "blob"
```

Comment:
Matthew had a comment that suggested setting `container_access_type` to "private" if the `azurestack_storage_account_sas` resource was available in terraform provider. I added this to the provider so maybe we need to add that too?